### PR TITLE
added zrevrangebylex in the list of available methods

### PR DESCRIPTION
**Pull request by abraovic**
Hi,

I've noticed that ZREVRANGEBYLEX method was missing from the list of available methods so I've added it.

Also I've added link to the official redis docs for that method.

Ante.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,4 @@ package.xml
 composer.lock
 experiments/
 vendor/
+.idea/
```
Comment (nrk) [inline .gitignore]: Can you remove this added line from `.gitignore` and restore the missing line feed?

**Comment by nrk** [comment]
Thanks @abraovic, please just fix `.gitignore` and I'll merge your PR later :+1:

**Comment by abraovic** [comment]
Thanks @nrk, it is fixed now

**Comment by nrk** [comment]
Thanks!

**Pull request merged**